backend/src/graphql/resolver/ContributionResolver.test.ts

```diff
@@ -671,6 +672,123 @@ describe('ContributionResolver', () => {
     })
   })
 
+  describe('denyContribution', () => {
+    describe('unauthenticated', () => {
+      it('returns an error', async () => {
+        await expect(
+          mutate({
+            mutation: denyContribution,
+            variables: {
+              id: 1,
+            },
+          }),
+        ).resolves.toEqual(
+          expect.objectContaining({
+            errors: [new GraphQLError('401 Unauthorized')],
+          }),
+        )
+      })
+    })
+
```

Comment:
Test that error is thrown, if you do not have admin rights. -> Login as Bibi, call denyContribution -> 401